kannon/master.py

```diff
@@ -56,6 +57,28 @@ def __init__(
         self.task_id_to_job_name: dict[str, str] = dict()
 
     def build(self, root_task: gokart.TaskOnKart) -> None:
+        # check all config file paths exists
+        # TODO: support multiple dynamic config files
+        remote_config_path = None
+        if self.dynamic_config_path:
+            logger.info("Handling dynamic config files...")
+            logger.info(f"Handling given config file {self.dynamic_config_path}")
+            # save configs to remote cache
+            if "TASK_WORKSPACE_DIRECTORY" not in os.environ:
+                raise ValueError("TASK_WORKSPACE_DIRECTORY is essential.")
+            remote_config_dir = os.path.join(os.environ["TASK_WORKSPACE_DIRECTORY"], "kannon", "conf")
```

Comment:
I think `dynamic_config_path` should be absolute path or relative path of `$PWD`. I think this default path is confusing.

Reply:
As mentioned in [my comment](https://github.com/m3dev/kannon/pull/39#issuecomment-1963271622), the purpose of the environment variable is to get the remote path. And we adopt `root_task.workspace_directory` for it, instead of the implementation you mentioned.